packages/common-styles/package.json

```
@@ -0,0 +1,46 @@
+{
+  "name": "@fluentui/common-styles",
+  "version": "0.1.0",
+  "description": "Common styling definitions for react components",
+  "main": "lib-commonjs/index.js",
+  "module": "lib/index.js",
+  "typings": "lib/index.d.ts",
+  "sideEffects": [
+    "lib/version.js"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/microsoft/fluentui"
+  },
+  "license": "MIT",
+  "scripts": {
+    "build": "just-scripts build",
+    "bundle": "just-scripts bundle",
+    "clean": "just-scripts clean",
+    "code-style": "just-scripts code-style",
+    "just": "just-scripts",
+    "lint": "just-scripts lint",
+    "start": "just-scripts dev:storybook",
+    "update-api": "just-scripts update-api"
```

Can be simplified as follows. You could potentially also remove `lint` since it won't do anything right now (though in the future, linting will probably include config files).
```suggestion
    "build": "just-scripts build",
    "clean": "just-scripts clean",
    "code-style": "just-scripts code-style",
    "just": "just-scripts",
    "lint": "just-scripts lint"
```